### DISCUSSION TITLE
[EASTER EGG] get that bark dog (game)

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#264] [New Features (not block requests)]
how to get to it: press the meme gif in explore 42 times
how it will work: you are your github pfp trying to chase down bark dog while he throws bones and blocks from bark into you, you have to use the arrow keys to catch up with the dog, if you reach him, the game resets and +1 round is added, making the game harder, if you lose your 5 lives by getting stuff thrown at you 5 times, the game is over and a high score gets added.

**Comment by raynepaws** [comment]
why github pfp and not bark account pfp

**Reply by Rubiidev-IPad** [reply]
because it's more recognizable
_(but you can make it use the bark account pfp if you want)_
edit: i don't think we have a default pfp yet

**Reply by raynepaws** [reply]
> because it's more recognizable

how do you plan on fetching a user's github profile picture

**Reply by Rubiidev-IPad** [reply]
some sort of simple pfp fetching api, and if the user is logged out use the default bark pfp after someone makes it
i think i already have one on @Rubiidev-18

**Reply by raynepaws** [reply]
> some sort of simple pfp fetching api

okay but how is it supposed to know what your github account is

**Reply by raynepaws** [reply]
> edit: i don't think we have a default pfp yet

![default profile picture](https://github.com/user-attachments/assets/798330b0-2a90-4bf5-bc1d-70effba6b1d2)

**Reply by Rubiidev-IPad** [reply]
oh

**Reply by mariocraft987** [reply]
> > some sort of simple pfp fetching api
> 
> okay but how is it supposed to know what your github account is

prb wont have  github acc pfp
